src/gui/modals/SetupDialog.cpp

```diff
@@ -1172,6 +1179,24 @@ void SetupDialog::audioInterfaceChanged(const QString & iface)
 }
 
 
+void SetupDialog::updateBufferSizeWarning(int value)
+{
+	QString text = "<ul>";
+	if((value & (value - 1)) != 0)  // <=> value is not a power of 2 (for value > 0)
```

Comment:
@JohannesLorenz  It looks like the LV2 plugins really see a maximum buffer size of 256 that you and jasp00 have been talking about. The [mod-pitchshifter](https://github.com/moddevices/mod-pitchshifter) plugins seem to work fine on all buffer sizes from 256 and up. Not just the ones that are a power of two. The correct range is then decided by the following test.

```cpp
if(((value & (value - 1)) != 0) && value < 256)
```

Do you want me to issue a PR for this with similar changes in the involved message strings and comments of do you want to look into the matter yourself?

Reply:
Essentially, the behavior I'm described [here](https://github.com/LMMS/lmms/pull/6484#issuecomment-1289317168) is the right one (column marked capo), just the warning for not being power of two is wrong on buffer sizes over 256.